result/arrow-spacing.js

```diff
@@ -0,0 +1,46 @@
+/*eslint-env es6*/
+
+// { "before": true, "after": true }
+(a) => {}
+
+// { "before": false, "after": false }
+(a)=>{}
```

Comment:
__[development][eslint]__
* Error - 'a' is defined but never used (no-unused-vars)
* Error - Missing space after => (arrow-spacing)